### PR TITLE
Use flags for log settings

### PR DESCRIPTION
**Pull request by tempoz**
Part of encapsulating these settings in the log lib is making Configure use the settings instead of other args.

---

**Version bump**: Minor

### CODE REVIEW

File: server/testutil/testenv/testenv.go
Comment (tylerwilliams) [inline server/testutil/testenv/testenv.go:40]: I understand you're just refactoring this code, so not a comment on your changes per-se, but:

How does this init() interplay with the one in log.go? They seem to be both doing the same thing?

Maybe instead of using init() here, we could have a sync.once in testEnv that configures the logger using some method like ConfigureForTesting(&opts{debug, shortFile})?

Comment (tylerwilliams) [reply server/testutil/testenv/testenv.go:40]: can you make this a sync.Once? otherwise CL LGTM

**Review by tylerwilliams — APPROVED** [review]

**Comment by tempoz** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1772?utm_source=stack-comment).


Job Reference: Qg74nVRI1LSaCBRkxMdN

**Comment by tempoz** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1772?utm_source=stack-comment).


Job Reference: IzK4IJkw70q5gig9wHik

**Pull request merged**
